src/main/java/org/cactoos/io/LoggingInputStream.java

```diff
@@ -213,12 +213,29 @@ public boolean markSupported() {
             msg = "Mark and reset NOT supported from %s";
         }
         this.logger.log(
-            this.logger.getLevel(),
+            this.level(),
             String.format(
                 msg,
                 this.source
             )
         );
         return supported;
     }
+
+    /**
+     * Retrieve {@code logger} level.
+     *
+     * @return Level of {@code logger}.
+     */
+    private Level level() {
```

Comment:
I think this part should be inside sticky scalar

Reply:
Done in [`954e7ee`](https://github.com/yegor256/cactoos/pull/715/commits/954e7eeebe7df20005d4d075254d38379fdd3251)